<launch>
	<node pkg="sscrovers_file_sequence_publisher" type="file_seq_pub_node" name="file_data_publisher" output="screen">

	<!-- frequency rate of node -->
	<param name="rate" value="0.5" />

	    <!-- name of output image topic -->
	    <param name="pub_image_topic_name" value="cam_image" />

	    <!-- name of output odometry topic -->
	    <param name="pub_odom_topic_name" value="odom" />

		<!-- path to location of images -->
	    <param name="path_to_images" value="$(find blob_analysis)/sample_images/" />

		<!-- prefix for images files name -->
	    <param name="image_file_name_prefix" value="frame_orig_" />

		<!-- path to location of trajectory -->
	    <param name="path_to_trajectory" value="$(find blob_analysis)/sample_images/" />
	    
	    <param name="trajectory_file_name" value="trajectory.fli" />
				
    	<param name="loop" value="true" />		

    	<param name="one_frame" value="-1" />		
	</node>
</launch>




